packages/xdl/src/detach/IosKeychain.ts

```diff
@@ -79,7 +87,9 @@ export async function cleanUpKeychains() {
       ['list-keychains'],
       { stdio: 'pipe' }
     );
-    const allKeychainsList = keychainsListRaw.match(/"(.*)"/g).map(i => i.slice(1, i.length - 1));
+    const allKeychainsList = (keychainsListRaw.match(/"(.*)"/g) || []).map(i =>
```

Comment:
Made a small change to runtime behavior here: instead of null pointer error we get an empty `allKeychainsList` if there's no match.

Reply:
Thanks!